graph/src/components/store.rs

```diff
@@ -660,6 +660,14 @@ pub trait Store: Send + Sync + 'static {
     /// Queries the store for a single entity matching the store query.
     fn find_one(&self, query: EntityQuery) -> Result<Option<Entity>, QueryExecutionError>;
 
+    /// Find the reverse of keccak256 for `hash` through looking it up in the
+    /// rainbow table.
+    fn find_ens_name(&self, _hash: &str) -> Result<Option<String>, QueryExecutionError> {
+        // The default implementation knows nothing about mapping keccak256 hashes
+        // to their preimage
+        Ok(None)
+    }
```

Comment:
I'm not a fan of having dummy default implementations, is this necessary?

Reply:
Neither am I, really. Fixing it.